docs/api/api_alternative.md

```diff
@@ -0,0 +1,289 @@
+| [AnalogOut](../apis/analogout.html) | &#10004; | &#10004; |
+| [DigitalIn](../apis/digitalin.html) | &#10004; | &#10004; |
+| [DigitalOut](../apis/digitalout.html) | &#10004; | &#10004; |
+| [DigitalInOut](../apis/digitalinout.html) | &#10004; | &#10004; |
+| [BusIn](../apis/busin.html) | &#10004; | &#10004; |
+| [BusOut](../apis/busout.html) | &#10004; | &#10004; |
+| [BusInOut](../apis/businout.html) | &#10004; | &#10004; |
+| [PortIn](../apis/portin.html) | &#10004; | &#10004;  |
+| [PortOut](../apis/portout.html) | &#10004; | &#10004;  |
+| [PortInOut](../apis/portinout.html) | &#10004; | &#10004;  |
+| [PwmOut](../apis/pwmout.html) | &#10004; | &#10004;  |
+| [InterruptIn](../apis/interruptin.html) | &#10004; | &#10004;  |
+
+### USB drivers
+
+The Mbed OS classes providing USB peripheral functionality, also known as USB components, inherit from USBDevice and provide specific USB peripherial functionality.
+
+| API | Full profile | Bare metal profile |
+| - | - | - |
+| [USBSerial](../apis/usb-apis.html) | &#10004; | &#10004;  |
+| [USBAudio](../apis/usbaudio.html) | &#10004; | &#10004;  |
+| [USBHID](../apis/ubshid.html) | &#10004; | &#10004;  |
+| [USBCDC](../apis/usbcdc.html) | &#10004; | &#10004;  |
+| [USBCDC_ECM](../apis/usbcdc-ecm.html) | &#10004; | &#10004;  |
+| [USBMSD](../apis/usbmsd.html) | &#10004; | &#10004; |
+| [USBMIDI](../apis/usbmidi.html) | &#10004; | &#10004;  |
+| [USBMouse](../apis/usbmouse.html) | &#10004; | &#10004;  |
+| [USBKeyboard](../apis/usbkeyboard.html) | &#10004; | &#10004;  |
+| [USBMouseKeyboard](../apis/usbmousekeyboard.html) | &#10004; | &#10004;  |
+
+### Other drivers
+
+The drivers are digital interfaces that allow your board to interface with a computer or external devices.
+
+| API | Full profile | Bare metal profile |
+| - | - | - |
+| [Timeout](../apis/timeout.html) | &#10004; | &#10004; |
+| [Watchdog](../apis/watchdog.html) | &#10004; | &#10004; |
+| [ResetReason](../apis/resetreason.html) | &#10004; | &#10004;  |
+| [Flash IAP](../apis/flash-iap.html) | &#10004; | &#10004;  |
+| [BufferedSerial](../apis/bufferedserial.html) | &#10004; | &#10004;  |
+| [UnbufferedSerial](../apis/unbufferedserial.html) | &#10004; | &#10004;  |
+| [SPI](../apis/spi.html) | &#10004; | &#10004;  |
+| [SPISlave](../apis/spislave.html) | &#10004; | &#10004;  |
+| [QuadSPI (QSPI)](../apis/quadspi-qspi.html) | &#10004; | &#10004; |
+| [I2C](../apis/i2c.html) | &#10004; | &#10004;  |
+| [I2CSlave](../apis/i2cslave.html) | &#10004; | &#10004;  |
+| [CAN](../apis/can.html)| &#10004; | &#10004;  |
+| [MbedCRC](../apis/mbedcrc.html) | &#10004; | &#10004;  |
+
+
+## Platform
+
+### Power
+
+<!--needs intro-->
+
+| API | Full profile | Bare metal profile |
+| - | - | - |
+| [Power management (sleep)](../apis/power-apis.html) | &#10004; | &#10004; |
+| [DeepSleepLock](../apis/deepsleeplock.html) | &#10004; | &#10004;  |
+| [Idle loop](../apis/idle-loop.html) | &#10004; | &#10004;  |
```

Comment:
This belongs to RTOS section (although it is related to power maangement since by default the idle loop invokes sleep manager to enter a sleep mode). Header and source files are under rtos/ so let's keep the documentation at the same place. Also this is NOT supported in bare metal (again confusion because the API was moved from rtos to platform table and so does not match the code)